hypervisor/include/dm/vpci.h

```diff
@@ -99,6 +99,9 @@ struct pci_vdev {
 	struct pci_msi msi;
 	struct pci_msix msix;
 
+	bool has_flr;
+	uint32_t pcie_capoff;
```

Comment:
Do we need the cache of those fields here? We can easily refer to pdev for these... Or do you mean modualization ?